feat: custom air conditioning

mjuhe opened the pull request:
<!-- If this PR closes an existing issue please add it using "Fixes #[issue_no]" here -->

## Summary
<!-- Please provide a quick summary of changes for this PR. -->
<!-- If required for your PR, please provide references to backup any documentation you are submitting. -->
This PR adds a page under Feature Guides to define the "Custom Air Conditioning System", mainly needed to support PR https://github.com/flybywiresim/aircraft/pull/8178 [Air Conditioning Failures]. 

My first contribution to docs so open to all comments and suggestions for improvement 😄. I'm not good at this kind of thing.

### Location
<!-- Please provide the original URL of the page modified or directory location here -->
docs/fbw-a32nx/feature-guides/custom-air-conditioning/

<!-- You may optionally provide your discord username, so that we may contact you directly about the issue. -->
Discord username (if different from GitHub): mjuhe

Valastiri commented:
Self note - Discuss options for a shorter title (for only the left navigation bar)

Valastiri commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,86 @@
+    Some airports provide low pressure conditioned air so aircraft don’t need to keep their APU on for extended periods of time in hot weather. This air is routed directly to the mixer unit, but this is not simulated yet in the A32NX.
+
+### Trim Air
+We’ve stablished that the packs deliver the minimum temperature selected on the overhead, and that the fans only recirculate cabin air (at whichever temperature this is in), so how do you achieve different temperatures in different cabin zones? With the trim air valves! These small valves “inject” hot air directly from the bleed air system to the air coming out of the mixer unit to fine tune the  ...
```
```suggestion
We’ve established that the packs deliver the minimum temperature selected on the overhead, and that the fans only recirculate cabin air (at whichever temperature this is in), so how do you achieve different temperatures in different cabin zones? With the trim air valves! 

These small valves “inject” hot air directly from the bleed air system to the air coming out of the mixer unit to fine tune the final temperature that gets delivered to each of the zones. This hot air can be switched on and off using the overhead panel.
```

minor typo but also suggest breaking up the long form paragraph (we have the space and suggestion in this review comment above). Makes for a nicer digital read.

Valastiri replied:
This can apply to other sections but figured would just include this one as a suggestion

Valastiri commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,86 @@
+- Cabin Fan 1 & 2
+- Lavatory and Galley Fan
+
+![EFB Air Conditioning Failures](../assets/feature-guides/air-conditioning/EFBFailures.JPG "EFB Air Conditioning Failures")
```
```suggestion
![EFB Air Conditioning Failures](../assets/feature-guides/air-conditioning/EFBFailures.JPG "EFB Air Conditioning Failures"){loading=lazy}
```
Prevents page load of image until in view

Valastiri commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,86 @@
+
+What you can expect to see when you activate them:
+
+- ACSC -> One lane fault: No effect as the second lane takes over. You will see a “COND CTL 1(2)-A(B) FAULT” ECAM warning (no chime) if you have engines off, otherwise no warning.
```
```suggestion
- ACSC -> One lane fault
	- No effect as the second lane takes over. You will see a “COND CTL 1(2)-A(B) FAULT” ECAM warning (no chime) if you have engines off, otherwise no warning.
```

suggest formatting the failures list into tiered bullet points for an easier read

mjuhe replied:
So I've tried making them a header now... Not sure if that looks better or worse than the format you suggested, happy to change it again, I'm definitely not an expert on this!

Valastiri commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,86 @@
+- Lav and Galley Fan: Cabin zone temperature sensors are ventilated by the air extracted by this fan. Cabin zone temperature regulation is lost. You will get a “COND LAV + GALLEY FAN FAULT” ECAM warning (single chime).
+!!! tip "If you like to see things burn you can fail the TAPRV open, then trigger an overheat and see what happens 😛."
+
+![ECAM Air Conditioning Failures](../assets/feature-guides/air-conditioning/air-cond-failures.jpg "ECAM Air Conditioning Failures")
```
```suggestion
![ECAM Air Conditioning Failures](../assets/feature-guides/air-conditioning/air-cond-failures.jpg "ECAM Air Conditioning Failures"){loading=lazy}
```

Valastiri reviewed (changes requested):
Couple things TODO

- I can optimize images for you as 400kb is rather large. Will get on this sometime today.
- Is there any resolution method to the failures you've described? (beyond turning the failure off).

I would hazard the following (in relation to resolutions):

- If yes -> maybe include basic resolution steps in an admonition or just via a new section
- If no -> maybe include an admonition to simply state we will be bringing resolution methods or etc. at a later time

mjuhe commented:
Thanks for the review @Valastiri !! I've implemented your comments and added some extra lines to break up paragraphs. I still think it looks like a lot of text and some graphs/images might help, but I don't have any ideas on what else to add 😄 

Regarding your comment about the failures resolution methods. There is no way to "fix" a failed item, but I've added a comment that says `Make sure you follow the ECAM actions to mitigate each failure!`. What do you think?

Valastiri commented:
> Thanks for the review @Valastiri !! I've implemented your comments and added some extra lines to break up paragraphs. I still think it looks like a lot of text and some graphs/images might help, but I don't have any ideas on what else to add 😄
> 
> Regarding your comment about the failures resolution methods. There is no way to "fix" a failed item, but I've added a comment that says `Make sure you follow the ECAM actions to mitigate each failure!`. What do you think?

heyo! I'll take a look tonight sorry was quite busy past couple of days. Yea that works! I wasn't entirely sure if there were should've given ya that option. That ECAM migitations sounds great!

If you don't mind me committing things I'll maybe break up the text a bit and even if it's long we can iterate down the line with more media if you'd like. or we can wait.

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+# Custom Air Conditioning System
+
+## System Description
+The Air Conditioning, Pressurization and Ventilation system (also known as Environmental Control System (ECS) or simply Air Conditioning System for convenience) regulates the temperature, pressure and airflow inside the aircraft.
```
```suggestion
The air Conditioning, pressurization and ventilation system (also known as Environmental Control System (ECS) or simply Air Conditioning System for convenience) regulates the temperature, pressure, and airflow inside the aircraft.
```

Valastiri replied:
@Sleinmaster question -- you missed the capital C for the first conditioning (intentional)? 

Additionally, keeping the capitilization in the second instance yes?

Valastiri replied:
went with "conditioning"

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+
+Its main functions are to maintain a high level of comfort for passengers and crew and to protect the aircraft systems that need it (like the avionics).
+
+Our aim at FBW is to model systems that match their real life counterpart one-to-one. Throughout this guide you will see items marked as “not yet implemented” or “coming soon”, that’s where we are still hard at work to bring you the most advanced Air Conditioning, Pressurization and Ventilation system you’ve ever seen in a home simulator.
```
```suggestion
Our aim at FBW is to model systems that match their real-life counterparts one-to-one. Throughout this guide, you will see items marked as “not yet implemented” or “coming soon”. In these cases, we are still hard at work to bring you the most advanced air conditioning, pressurization and ventilation system you’ve ever seen in a home simulator.
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+
+Our aim at FBW is to model systems that match their real life counterpart one-to-one. Throughout this guide you will see items marked as “not yet implemented” or “coming soon”, that’s where we are still hard at work to bring you the most advanced Air Conditioning, Pressurization and Ventilation system you’ve ever seen in a home simulator.
+
+Please note this guide doesn’t go through all conditions that trigger specific situations, you are encouraged to discover these details yourself as you fly along the open skies!
```
```suggestion
Please note, this guide doesn’t go through all conditions that trigger specific situations; you are encouraged to discover these details yourself as you fly along the open skies!
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+Please note this guide doesn’t go through all conditions that trigger specific situations, you are encouraged to discover these details yourself as you fly along the open skies!
+
+## Cabin Simulation
+The first step in order to be able to model the Air Conditioning system is to simulate the air inside of a virtual cabin.
```
```suggestion
The first step in order to be able to model the air conditioning system is to simulate the air inside a virtual cabin.
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+## Cabin Simulation
+The first step in order to be able to model the Air Conditioning system is to simulate the air inside of a virtual cabin.
+
+Microsoft Flight Simulator doesn’t provide a default implementation for it, so we’ve developed our own. For every “tick” in MSFS, the A32NX runs a separate fluid simulation inside of the fuselage that accurately mimics air behaviour in real life.
```
```suggestion
Microsoft Flight Simulator doesn’t provide a default implementation for it, so we’ve developed our own. For every “tick” in MSFS, the A32NX runs a separate fluid simulation inside the fuselage that accurately mimics air behaviour in real life.
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+
+Microsoft Flight Simulator doesn’t provide a default implementation for it, so we’ve developed our own. For every “tick” in MSFS, the A32NX runs a separate fluid simulation inside of the fuselage that accurately mimics air behaviour in real life.
+
+Some of the details and features of this simulation include:
```
```suggestion
Some details and features of this simulation include:
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+The air conditioning system of the A320 is better understood by following an air particle through its journey from the bleed system to the cabin.
+
+### Air Conditioning Packs
+Air from the bleed system is hot. It comes directly from the engines or APU and it needs to be cooled down before it can be delivered to the cabin. This happens in the packs, which are heat exchangers that cool and dry the air.
```
```suggestion
Air from the bleed system is hot. It comes directly from the engines or APU, and it needs to be cooled down before it can be delivered to the cabin. This happens in the packs, which are heat exchangers that cool and dry the air.
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+### Air Conditioning Packs
+Air from the bleed system is hot. It comes directly from the engines or APU and it needs to be cooled down before it can be delivered to the cabin. This happens in the packs, which are heat exchangers that cool and dry the air.
+
+The packs regulate their output temperature to target the minimum temperature selected on the overhead panel. They can be turned off with their respective pushbutton on the overhead panel, but their intake valve also closes in a few other situations, like when the engines are off or when the ditching pushbutton is pressed in.
```
```suggestion
The packs regulate their output temperature to target the minimum temperature selected on the overhead panel. They can be turned off with their respective push-button on the overhead panel, but their intake valve also closes in a few other situations, like when the engines are off or when the ditching push-button is pressed in.
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+The packs regulate their output temperature to target the minimum temperature selected on the overhead panel. They can be turned off with their respective pushbutton on the overhead panel, but their intake valve also closes in a few other situations, like when the engines are off or when the ditching pushbutton is pressed in.
+
+!!! info "Not Yet Implemented"
+    The inner workings of the packs haven’t been modelled yet in the A32NX but they are in the works!
```
```suggestion
    The inner workings of the packs haven’t been modelled yet in the A32NX, but they are in the works!
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+They are housed in the underbelly of the aircraft and are controlled by a pushbutton on the overhead panel. 
+
+### Mixer Unit
+This is where the air from the packs and the cabin fans gets mixed (the name is pretty self explanatory!).
```
```suggestion
This is where the air from the packs and the cabin fans gets mixed (the name is pretty self-explanatory!).
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+This is where the air from the packs and the cabin fans gets mixed (the name is pretty self explanatory!).
+
+!!! info "Not Yet Implemented"
+    Some airports provide low pressure conditioned air so aircraft don’t need to keep their APU on for extended periods of time in hot weather. This air is routed directly to the mixer unit, but this is not simulated yet in the A32NX.
```
```suggestion
    Some airports provide low pressure conditioned air, so aircraft don’t need to keep their APU on for extended periods of time in hot weather. This air is routed directly to the mixer unit, but this is not simulated yet in the A32NX.
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+### Air Conditioning System Controller (ACSC)
+The Air Conditioning System Controller is a dual channel, dual lane system that controls all components of the Air Conditioning System. 
+
+When a zone temperature is selected on the overhead, it calculates the temperature demand and air flow required in the cabin outlets to achieve it. It send signals to all the moving parts of the system (described above!) to achieve the right balance and meet the temperature and flow demands.
```
```suggestion
When a zone temperature is selected on the overhead, it calculates the temperature demand and airflow required in the cabin outlets to achieve it. It sends signals to all the moving parts of the system (described above!) to achieve the right balance and meet the temperature and flow demands.
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+When a zone temperature is selected on the overhead, it calculates the temperature demand and air flow required in the cabin outlets to achieve it. It send signals to all the moving parts of the system (described above!) to achieve the right balance and meet the temperature and flow demands.
+
+!!! tip "Did You Know?"
+    The ACSC is an Airbus optional system for the A320. Some A320s come with individual local controllers for each of the elements (temperature, flow, trim air…). We’ve modelled the A32NX to come equipped with an ACSC, because why not!
```
```suggestion
    The ACSC is an Airbus optional system for the A320. Some A320s come with individual local controllers for each of the elements (temperature, flow, trim air…). We’ve modelled the A32NX to come equipped with an ACSC because why not!
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+#### ACSC -> Two lane fault
+- The related pack is lost, and the hot air pressure-regulating valve and trim air valves close. You will get a “AIR PACK 1(2) FAULT” ECAM warning (single chime). Temperature regulation is lost.
+#### Trim Air Pressure Regulating Valve
+- The “hot air” valve is stuck in position. If it fails open this has no effect. If it fails closed temperature optimisation is lost. You will only get a “HOT AIR FAULT” when the valve disagrees with the commanded position (note it takes a while to get the warning!). The FAULT light illuminates on the overhead.
```
```suggestion
- The “hot air” valve is stuck in position. If it fails open, this has no effect. If it fails closed, temperature optimisation is lost. You will only get a “HOT AIR FAULT” when the valve disagrees with the commanded position (note it takes a while to get the warning!). The FAULT light illuminates on the overhead.
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+#### Trim Air Pressure Regulating Valve
+- The “hot air” valve is stuck in position. If it fails open this has no effect. If it fails closed temperature optimisation is lost. You will only get a “HOT AIR FAULT” when the valve disagrees with the commanded position (note it takes a while to get the warning!). The FAULT light illuminates on the overhead.
+#### Trim Air System High Pressure
+- Hot air pressure increases but it has no observable effect. You will get a “COND TRIM AIR SYS FAULT” and “TRIM AIR HI PR” ECAM message (no chime).
```
```suggestion
- Hot air pressure increases, but it has no observable effect. You will get a “COND TRIM AIR SYS FAULT” and “TRIM AIR HI PR” ECAM message (no chime).
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+#### [CKPT/FWD/AFT] Duct Overheat
+- The duct temperature rises above 88C momentarily. The TAPRV and trim air valves close. Temperature optimization is lost. You will get a “COND FWD CAB/AFT CAB/CKPT DUCT OVHT” ECAM warning (single chime).
+#### Cabin Fan
+- One cabin fan failure reduces the recirculated flow, if both fail there is no air recirculation. You can see an amber “FAN” message appear in the COND page. You will get a “COND L+R CAB FAN FAULT” alert if both fail.
```
```suggestion
- One cabin fan failure reduces the recirculated flow, if both fail, there is no air recirculation. You can see an amber “FAN” message appear on the COND page. You will get a “COND L+R CAB FAN FAULT” alert if both fail.
```

Sleinmaster commented on docs/fbw-a32nx/feature-guides/custom-air-conditioning.md:
```diff
@@ -0,0 +1,112 @@
+#### Lav and Galley Fan
+- Cabin zone temperature sensors are ventilated by the air extracted by this fan. Cabin zone temperature regulation is lost. You will get a “COND LAV + GALLEY FAN FAULT” ECAM warning (single chime).
+
+!!! tip "If you like to see things burn you can fail the TAPRV open, then trigger an overheat and see what happens 😛."
```
```suggestion
!!! tip "If you like to see things burn, you can fail the TAPRV open, then trigger an overheat and see what happens 😛."
```

Sleinmaster reviewed (commented):
Some spell-checking, will have a more in-depth read some other time when the duties allow it.

Valastiri commented:
I might review this tonight and commit changes to get this ready for merge

Valastiri commented:
check build

Valastiri commented:
will solve with rebase shortly with optimized images + some additional changes
